eya_def_tools/eya_def_tools/data_models/dataset.py

```diff
@@ -116,6 +183,12 @@ class DatasetDimension(StrEnum):
     MONTH = auto()
     YEAR = auto()
 
+    START_DATE = auto()
```

Comment:
The binning dimensions `start_date` and `end_date` or `start_datetime` and `end_datetime` have been introduced to allow flexible specification of periods when defining bins. This can for example be used to express the first ten years of operation.